tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph/graph_component.ts

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'graph-component',
```

Comment:
If possible, try to use the same name as the container.

Reply:
IIUC, this will involve changing names such as graph-component to tf-debugger-v2-graph-component. Existing components and containers in plugins/debugger_v2 already have inconsistent names like this. I'll send another PR to make them all consistent later.